src/libcip54.ts

```diff
@@ -629,6 +1066,32 @@ export const getMintTx = async (
   return mintTx.rows[0];
 };
 
+/**
+* @description This function retrieves the CIP-68 metadata for a given unit by
+* querying the PostgreSQL database using GraphQL. It checks if the metadata is cached
```

Comment:
I think this is another random AI artifact - we're not using GraphQL at all here, this is just standard SQL